eng/common/scripts/add-issue-assignees.ps1

```diff
@@ -0,0 +1,55 @@
+param(
+    [Parameter(Mandatory = $true)]
+    $RepoOwner,
+
+    [Parameter(Mandatory = $true)]
+    $RepoName,
+
+    [Parameter(Mandatory = $false)]
+    $GitHubUsers = "",
+
+    [Parameter(Mandatory = $true)]
+    $IssueNumber,
+  
+    [Parameter(Mandatory = $true)]
+    $AuthToken
+)
+
+function AddMembers($memberName, $additionSet) {
+  $headers = @{
+    Authorization = "bearer $AuthToken"
+  }
+  $uri = "https://api.github.com/repos/$RepoOwner/$RepoName/issues/$IssueNumber"
+  $errorOccurred = $false
+
+  foreach ($id in $additionSet) {
+    try {
+      $postResp = @{}
+      $postResp[$memberName] = @($id)
+      $postResp = $postResp | ConvertTo-Json
+
+      Write-Host $postResp
+      $resp = Invoke-RestMethod -Method Patch -Headers $headers -Body $postResp -Uri $uri -MaximumRetryCount 3
+      $resp | Write-Verbose
+    }
+    catch {
```

Comment:
Is there a reason not to use ```ErrorAction = Stop``` in these scripts rather than having to pass failure state back up to the caller?

Reply:
For reusable functions I don't think it is a good idea to fail fast as they might not know the full context where they are being used.